corehq/apps/app_manager/templates/app_manager/v1/partials/app-settings.html

```diff
@@ -172,6 +172,14 @@
 "></inline-edit>
 
 <div class="tab-pane" id="commcare-settings">
+    {% if app.get_doc_type == 'LinkedApplication' %}
+        <p> Your app is at version {{ app.version }}. The master app is at version {{ master_version }}. </p>
+        <a class="btn btn-primary"
+            href="{% url 'pull_master_app' domain app.id %}">
+            {% trans 'Pull Master App' %}
```

Comment:
This label seems pretty git-esque, which is likely not intuitive to most of our end-users. What about changing it to something like *Update from master app* or *Get changes from master app*?